packages/test-helpers/src/index.js

```diff
@@ -12,11 +14,14 @@ const mocha = require('./mocha')
 const node = require('./node')
 const numbers = require('./numbers')
 const time = require('./time')
+const web3 = require('./web3')
 
 module.exports = {
   injectArtifacts,
   injectWeb3,
   ...addresses,
+  ...aragonOS,
+  ...asserts,
```

Comment:
Ah, I had actually wanted to avoid re-exporting these directly from `index.js`, especially not destructured.

I was hoping to get something like:

```
import { ANY_ENTITY } from '@aragon/contract-helpers-test/aragon-os'
```

For the user, but this turned out difficult to do.

-----

Would appreciate any other suggestions, but otherwise I wonder if we should keep it for now as:

```
import { ANY_ENTITY } from '@aragon/contract-helpers-test/aragon-os/src/aragon-os'
```

Or export the entire object out, rather than destructuring it, so it looks like this to the user:

```
import { aragonOS } from '@aragon/contract-helpers-test/aragon-os'
const { ANY_ENTITY } = aragonOS
```

Reply:
Is this to optimize? Does it have a great impact in performance?
I did it this way to not have to remember the path of those methods, which is less convenient.

Reply:
Not at all, I wanted to provide some structure to group some utilities / asserts as not all of them may be useful in a vanilla project (e.g. Staking).